app/assets/stylesheets/dashboards.scss

```diff
@@ -24,3 +24,28 @@
 .bootstrap_toggle_spacing .checkbox {
   margin-top: 0px;
 }
+
+.pt-phone,.pt-LMP {
+  width: 100px;
+  white-space: nowrap;
+  padding-right: 5px;
+}
+
+.pt-recent-notes {
```

Comment:
I think reducing the size of the note is a very good idea.  What's that controlled by?  Have users expressed that it's useful to them?  Because if not I would consider getting rid of it if we're displaying so little text it's not helpful to them.